service/flannelconfig/v3/resource_set.go

```diff
@@ -213,11 +213,6 @@ func NewResourceSet(config ResourceSetConfig) (*framework.ResourceSet, error) {
 		if key.VersionBundleVersion(customObject) == VersionBundle().Version {
 			return true
 		}
-		// TODO remove this hack with the next version bundle version or as soon as
-		// all certconfigs obtain a real version bundle version.
-		if key.VersionBundleVersion(customObject) == "" {
-			return true
-		}
```

Comment:
@xh3b4sd Is this removal correct in here?

Reply:
Since this is a new version with a new version bundle we only want to reconcile the ones having the specific version we support here. The others are still managed by the older version. So yes, this here should be good.

Reply:
Yes, and this is cool how smooth the migration is thanks to the version bundles.